crypto/BRCryptoWalletManagerClient.h

```diff
@@ -45,89 +45,89 @@ extern "C" {
 
     typedef void
         (*BRCryptoCWMEthGetEtherBalanceCallback) (BRCryptoCWMClientContext context,
-                                                  BRCryptoWalletManager manager,
-                                                  BRCryptoCWMClientCallbackState callbackState,
-                                                  const char *network,
-                                                  const char *address);
+                                                  OwnershipGiven BRCryptoWalletManager manager,
+                                                  OwnershipGiven BRCryptoCWMClientCallbackState callbackState,
+                                                  OwnershipKept const char *network,
+                                                  OwnershipKept const char *address);
```

Comment:
I usually consider  `const char *` as implicitly OwnershipKept but being explicit is better.

Reply:
Didn't know how far down we wanted to go with this.